### ISSUE TITLE
Remove redirect from scheduling url

### ISSUE THREAD

**Issue by outerpress** [department-of-veterans-affairs/va.gov-team#72066]
## Description
_What details are necessary for understanding the specific work or request tracked by this issue?_

This work makes it possible to send a user directly to the beginning of the scheduling flow from the MHV on VA.gov home page.

This ticket does not add this link to the MHV on VA.gov home page. That will be done by the MHV on VA.gov team.

VAOS scheduling flow URL: `https://va.gov/my-health/appointments/schedule/type-of-care`
MHV on VA.gov home URL: `https://va.gov/my-health/`

---

## Acceptance Criteria
Background: the `toggle` toggle is on

Given 
When 
Then 
And page content matches the [copy doc](#) 
And page design matches the design spec

Given 
When 
Then 

### Design Assets
All spec copy is FPO. Use copy in copy doc.

## Definition of Done
- [ ] All acceptance criteria are met
- [ ] Documentation is updated

**Comment by outerpress** [comment]
@simiadebowale @JRRoof here's the ticket for deep-linking to the scheduling flow - you'll need to add ACs

**Comment by simiadebowale** [comment]
@ldelacosta - this is ready for review

**Comment by outerpress** [comment]
@simiadebowale @ldelacosta here's what I tested and the results. How else should I test this?
- I was able to navigate directly to https://staging.va.gov/my-health/appointments/schedule/type-of-care as a logged in user

Note: We talked about these being related, so just calling them out in case they should have been resolved too:
- When I refreshed a page in the scheduling flow, it still returned me to the VAOS home page instead of refreshing the current page. See #40601
- The link to the schedule flow from the empty state still doesn't work. See #71713.
